### PR TITLE
Fix underflow issue with Oracle JDBC

### PR DESCRIPTION
**Pull request by ngupta0205**
## Problem
https://confluentinc.atlassian.net/browse/RCCA-12408

## Solution
https://github.com/jOOQ/jOOQ/issues/7548

##### Does this solution apply anywhere else?
- [ ] yes
- [x ] no

##### Testing done:
- [ ] Unit tests
- [x ] Integration tests
- [ ] System tests
- [ ] Manual tests

### CODE REVIEW

**Comment by ManasjyotiSharma** [comment]
Thanks for writing the PR @ngupta0205. Couple of points around testing:

1. Could you update the PR description with the testing strategy you followed (_to verify the fix_)?
2. If not already there; Could you add relevant tests into OracleDatatypeIT for the newly added FLOAT32/FLOAT64 scenarios?

**Comment by ManasjyotiSharma** [comment]
@ngupta0205 additionally I see that the jenkins build stage is failing - can you please look at that?

**Comment by ManasjyotiSharma** [comment]
BTW @ngupta0205 this seems more of a bug fix so a patch version update should suffice. Can you please change the target branch accordingly instead of master?

**Comment by ngupta0205** [comment]
> BTW @ngupta0205 this seems more of a bug fix so a patch version update should suffice. Can you please change the target branch accordingly instead of master?

@ManasjyotiSharma Added the IT for this and changed the base branch to 10.7.x. PTAL.

File: pom.xml
Comment (ManasjyotiSharma) [inline pom.xml:1]: @ngupta0205 I believe none of these pom changes are required; can you please check once and revert this file?

Comment (ngupta0205) [reply pom.xml:1]: <scope> change is needed. The other ones got it when I changed the target branch to 10.7.x.

Comment (ngupta0205) [reply pom.xml:1]: Removed the changes other than "scope" change.

**Comment by ManasjyotiSharma** [comment]
@ngupta0205 LGTM; thanks for writing the PR!

**Review by ManasjyotiSharma — APPROVED** [review]

**Pull request merged**